app/scripts/modules/kubernetes/v2/kubernetes.v2.module.ts

```diff
@@ -0,0 +1,25 @@
+import {module} from 'angular';
```

Comment:
Add some space: `{ module }`. I can show you the IntelliJ setting for this.

Reply:
Found it